kolibri/core/content/management/commands/exportcontent.py

```diff
@@ -117,17 +120,23 @@ def export_file(self, f, data_dir, overall_progress_update):
         if os.path.isfile(dest) and os.path.getsize(dest) == f["file_size"]:
             overall_progress_update(f["file_size"])
             return
-
         copy = transfer.FileCopy(srcpath, dest, cancel_check=self.is_cancelled)
-
         with copy, self.start_progress(
             total=copy.total_size
         ) as file_cp_progress_update:
             try:
                 for chunk in copy:
                     length = len(chunk)
+                    self.exported_size = self.exported_size + length
+                    print(self.exported_size)
                     overall_progress_update(length)
                     file_cp_progress_update(length)
             except transfer.TransferCanceled:
+                job = get_current_job()
+                if job:
+                    job.extra_metadata["file_size"] = self.exported_size
+                    job.extra_metadata['total_resources'] = 0
```

Comment:
I think being able to give a meaningful number here would be the best result - I can give some guidance on how to estimate these using some temporary annotation techniques, similar to what we use for importability.